open-problems.md

```diff
@@ -1,5 +1,18 @@
-# Open research problems
+---
+title: Open Research Problems
+---
 
-This page is a stub. It will be filled out with open research problems.
+# Open Research Problems
 
-For now, please take a look at our open issues. For example in the [vacp2p/research repo](https://github.com/vacp2p/research/issues).
+We are seeking to collaborate with academia in the following topic areas.
```

Comment:
Not only academia, also independent researchers - most people in the web3 space are more independent

The phrasing sounds a a bit too... dreary? Maybe say we have a lot of research areas and some specific open problems, and are excited to see contributions from people (affiliated or not)